src/utils/sanitizeProps.js

```diff
@@ -0,0 +1,15 @@
+export const omit = (obj, keysToRemove) => {
```

Comment:
we already have `lodash-compat` in the project so let's use `lodash-compat/object/omit` instead.

We __do__ want a bootstrap specific helper though, that can go in `uitls/bootstrapUtils`. It can simply wrap lodash omit, but also remove the boostrap specific props.

```js
function omitProps(props, keys) {
  return omit(props, [...keys, 'bsRole', 'bsStyle', 'bsClass'])
}
```